scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -83,16 +83,16 @@
           <th style="width: 10%;">Status <a href="" id="{{group_name}}_status" onclick="updateArgs('{{group_name}}','sort=status','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='status' else 'grey'}};background-color:white;">
             <i class="fa fa-sort{{'-'+sort_order if sort_by=='status' }}" aria-hidden="true" data-bs-toggle="tooltip" title="Sort cases by 'status'"></i>
           </a></th>
-          <th style="width: 15%;">Analysis date <a href="" id="{{group_name}}_analysis_date" onclick="updateArgs('{{group_name}}','sort=analysis_date','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='analysis_date' else 'grey'}};background-color:white;">
+          <th style="width: 12%;">Analysis date <a href="" id="{{group_name}}_analysis_date" onclick="updateArgs('{{group_name}}','sort=analysis_date','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='analysis_date' else 'grey'}};background-color:white;">
             <i class="fa fa-sort{{'-'+sort_order if sort_by=='analysis_date' }}" aria-hidden="true" data-bs-toggle="tooltip" title="Sort by analysis date"></i>
           </a></th>
-          <th style="width: 15%;">Link</th>
-          <th style="width: 15%;">Default panel</th>
-          <th style="width: 5%;">Sequencing</th>
-          <th style="width: 10%;">Track <a href="" id="{{group_name}}_track" onclick="updateArgs('{{group_name}}','sort=track','{{sort_option}}');" class="badge text-body" style="color:{{'black' if sort_by=='track' else 'grey'}};background-color:white;">
+          <th style="width: 14%;">Link</th>
+          <th class="w-25">Default panel</th>
```

Comment:
```suggestion
          <th style="width 25%">Default panel</th>
```

Reply:
For consistency with the other cells?

Reply:
Hihi, but it would be inconsistent with the other ´w-25´. Either way is perfectly fine; I would have gone with the general, but your suggestion is lacking a colon, so keeping the w-25s for now. 😉